### DISCUSSION TITLE
Translate a complete space with GitBook AI and keep translations up to date after changes

### DISCUSSION THREAD

**Discussion by techfreak01** [GitbookIO/community#704] [Feature Requests]
I would like to create my product documentation in one base language in one space and use AI to translate it into multiple other languages (one space per language containing the same pages).

And when I later update the documentation in the space with the base language (edit existing or add new pages), AI should also update the translated spaces with the same changes.

**Comment by toscanelli** [comment]
This is what exactly we need. Is there any feature to translate a complete space creating a new one?

**Comment by saratandowsky** [comment]
You can already translate a space with AI docs [here](https://docs.gitbook.com/content-editor/editing-content/write-and-edit-with-ai). We are working on auto-translation though where you'll be able to set a source space and have it translated into other languages with AI.

**Comment by ElijahYerington** [comment]
Adding on to the previous comments. Happy to hear it's being worked on, translating a page as a one-time event is useful, but what would really be great is if we could have a source space with translations that sync to edits made to the source.
